about.html

```diff
@@ -0,0 +1,193 @@
+            </div>
+            
+          </li>
+          <li>
+              <a href="index.html" id="pop-buttons-1">Home</a>
+          </li>
+          <li>
+              <a href="#" class="" id="pop-buttons-2">Events</a>
+          </li>
+          <li>
+              <a href="#" class="" id="pop-buttons-3">News</a>
+          </li>
+      </ul>
+      
+  </div>
+
+  <main>
+    <section class="welcoming-part pb-3">
+      <div class="px-5 pt-5 d-md-none">
+        <button class="menu" id="menu-id" type="button">
+            <p class="menu-icon"></p>
+            <p class="menu-icon"></p>
+            <p class="menu-icon"></p>
+        </button>
+      </div>
+      <div class="container pt-5 text-center">
+        <h2 class="welcome-msg-2">Concert Hub. 
+          A place to get your ticket for any concerts.</h2>
+      </div>
+
+      <div class="container text-center">
+        <p class="test p-2 bg-light "> Lorem ipsum dolor sit amet, 
+          consectetur adipiscing elit, 
+          sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+           Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolore
+           </p>
+      </div>
+      <div class="program-schedule container text-center">
+        <p>Please contact us per Email for any further questions about Concert-Hub 2021 <strong class="email-tag"> test.test@gmail.com</strong> </p>
+      </div>
+    </section>
+
+
+   
+
+    <section class="current-concert p-5 text-center">
+      <h2 class="head-2 pb-1">CONCERT-HUB 2021 LOGO COMPETITION</h2>
+      
+      <div class="line mb-5"></div>
+
+      <div class="container text-center py-0 mb-5">
+          <div class="col-md  m-3 ">
+            <span class="logo curnt-card py-3" id="logo-3">Concert-Hub</span>
+          </div>
+      </div>
+    </section>
+
+
+
+    <section class="pt-5 text-center">
+      <h2 class="head-2 pb-1">Lorem ipsum dolor sit amet consectetur adipisicing elit.</h2>
+      
+      <div class="line "></div>
+
+      <div class="container p-0">
+        <div class="row">
+          
+          <div class="col-md mt-3" style=""> 
+            <img src="./images/versions.jfif" alt="" class="m-2" style="width: 20rem; height: 15rem; border-radius: 5px;">
+            <h6 class="program-title p-3">2020</h6>
+          </div>
+
+          <div class="col-md mt-3">
+            <img src="./images/versions.jfif" alt="" class="m-2" style="width: 20rem; height: 15rem; border-radius: 5px;">
+            <h6 class="program-title p-3">2021</h6>
+          </div>
+        </div>
+      </div>
+
+    </section>
+
+
+    
+  </main>
+  
+  <footer class="bg-dark navbar-dark pt-3" style="height: 150px;">
+    
+    <div class=" d-flex justify-content-around m-5 ">
+      <span class="logo" id="logo-4">Concert-Hub</span>
+      <small class="rights">© Concert-Hub 2021. Some Rights</small>
+    </div>
+  </footer>
```

Comment:
- [ ] On desktop view the footer has a dark background whereas on mobile view the footer has a white background. Please apply the changes on both pages.